sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1431,4 +1431,30 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("insert data to a table which has altered the table location " +
+    "to an not exist location should success") {
+    withTable("t") {
+      withTempDir { dir =>
+        spark.sql(
+          s"""create table t(a string, b int)
```

Comment:
General style suggestions. Please use upper case for SQL keywords. For example, in this SQL statement can be improved to
```
CREATE TABLE t(a STRING, b INT)
USING parquet
OPTIONS(path "xyz")
```